changelog.d/20240319_140241_andrew.bonnell_add_config_user_filter.md

```diff
@@ -0,0 +1 @@
+- [Feature] Introduces the CONFIG_USER Filter. (by @abonnell)
```

Comment:
nit: You can add some more details of the filter here. 1-2 liner explaining what can be expected from this. I understand the info is there in docstring but changelog are important for the larger community.

Reply:
https://github.com/overhangio/tutor/pull/1021/commits/d0b3229f16bb612582bb431a418c0ff8b5ea775c

Updated changelog message to be a bit more descriptive